Assets/.keep

```diff
@@ -0,0 +1 @@
+Workaround for CloudBuild automation

```

Comment:
See above. CB bails out if the repo doesn't look like a Unity project. We can probably remove these folders and the ucb_preload script for now since we still can't actually use UCB for testing.